int16 velStepper1
int16 velStepper2
int16 velStepper3
int16 velStepper4
int16 velStepper5
int16 velStepper6